src/prisma/seeds/utils.ts

```diff
@@ -25,3 +25,41 @@ export const seedTable = async (
   console.log(`Created ${creation?.count} records in ${name} table`);
   return await prisma[name].findMany();
 };
+
+export function getRandomCoordinates(
+  lat: number,
+  lon: number,
+  radiusInMeters: number,
+) {
+  const radiusInDegrees = radiusInMeters / 111300;
+
+  const u = Math.random();
+  const v = Math.random();
+  const w = radiusInDegrees * Math.sqrt(u);
+  const t = 2 * Math.PI * v;
+  const deltaLat = w * Math.cos(t);
+  const deltaLon = (w * Math.sin(t)) / Math.cos(lat * (Math.PI / 180));
+
+  const newLat = lat + deltaLat;
+  const newLon = lon + deltaLon;
+
+  return { latitude: newLat, longitude: newLon };
+}
```

Comment:
probably can do it with [fakerjs](https://fakerjs.dev/api/location.html#nearbygpscoordinate)

Reply:
Nop, the point is to have random coordinate in a specific perimeter to be able to test the map. Faker returns locations that are too random and therefore hard to see in a zoomed view on the map.